s3/s3.go

```diff
@@ -91,6 +91,7 @@ type S3ClientOpts struct {
 	RoleSessionName string
 	UseSDKCreds     bool
 	EncryptOpts     EncryptOpts
+	EnableMD5       bool
```

Comment:
Since the [api description in minio](https://min.io/docs/minio/linux/developers/go/API.html) is
> Specify if you’d like to send content-md5 header with PutObject operation. Note that setting this flag will cause higher memory usage because of in-memory md5sum calculation.

I probably keep this variable as `SendContentMd5`, it describe what the variable is doing more accurately